package.json

```diff
@@ -50,6 +50,7 @@
     "node-uuid": "^1.4.3",
     "page": "^1.6.4",
     "postcss": "^5.0.6",
+    "query-string": "^3.0.1",
```

Comment:
Its [simpler](https://github.com/sindresorhus/query-string/blob/master/index.js) and appears to be better maintained than the 'querystring' implementation which Browserify is using by default (`querystring-es3`).